client/knowledge_base_search/src/containers/SearchContainer/SearchContainer.js

```diff
@@ -19,6 +20,27 @@ class SearchContainer extends Component {
     }
   }
 
+  getFiveRandomQueries() {
+    const totalQueries = randomQueries.length;
```

Comment:
why not 
1. make a copy of the array with `randomQueries.slice(0)`
1. use a for-loop to iterate over `randomQueries.length` times
1. create a new array by removing the `Math.floor(Math.random() * randomQueries.length)` using `array.splice(index, 1)`
1. use the new array as your random queries